docs/root/configuration/http_filters/jwt_authn_filter.rst

```diff
@@ -56,32 +56,49 @@ Remote JWKS config example
 
 .. code-block:: yaml
 
-  providers:
-    provider_name1:
-      issuer: https://example.com
-      audiences:
-      - bookstore_android.apps.googleusercontent.com
-      - bookstore_web.apps.googleusercontent.com
-      remote_jwks:
-        http_uri:
-          uri: https://example.com/jwks.json
-          cluster: example_jwks_cluster
-        cache_duration:
-          seconds: 300
+          http_filters:
```

Comment:
done

Reply:
? I meant to de-indent all this block, it will be done by sphinx anyway:
https://121193-65214191-gh.circle-artifacts.com/0/source/generated/docs/configuration/http_filters/jwt_authn_filter.html#configuration

Reply:
I am not entirely clear on what you mean by "de-indent".
Please review my change and if there is something wrong, could explain in detail what you mean.